tests/mbis-1/input.dat

```diff
@@ -76,7 +76,7 @@ set {
 }
 
 e, wfn = energy('hf/cc-pvdz', return_wfn=True)
-oeprop(wfn, 'MBIS_CHARGES', title='H20 SCF')
+oeprop(wfn, 'MBIS_CHARGES', "MBIS_VOLUME_RATIOS", title='H20 SCF')
```

Comment:
Two things: 
- it would make sense to check that `oeprop(wfn, 'MBIS_CHARGES')` works independently from running `oeprop(wfn, 'MBIS_VOLUME_RATIOS')`; either this test, or `mbis-2` below are good candidates.
- the quotation marks are inconsistent - is this also because of `black`?

Reply:
good idea -- I've split `mbis-1` up now.

the quotes are just my style has changed from single to double to placate `black`. the driver codebase will get a `black` run someday, I expect, so I often convert any lines I touch for other reasons to minimize churn at format time. these tests won't likely get that treatment.

Reply:
The reference test outputs are missing, but apart from that it looks OK so I've approved it.